CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 
-set(CMAKE_C_COMPILER "clang")
-set(CMAKE_CXX_COMPILER "clang++")
+#set(CMAKE_C_COMPILER "cc")
+#set(CMAKE_CXX_COMPILER "c++")
```

Comment:
But I REALLY want LLVM ;) 

Let me think of the best way to allow a compiler override.